# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsGfxScrollFrame.cpp
# Commit: a269192c5d30
# Full Hash: a269192c5d3037c7841d30219516548a6fa8db1a
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-06-27 09:34:48
# Regressor Bug: 1560237
# File Overlap Count: 1
# Description:
#   Bug 1560237 - Rename ScrollFrameHelper::GetFrameForScrollSnap to ScrollFrameHelper::GetFrameForStyle. r=botond
#   
#   The function will also be used for `scroll-behavior` property in a later patch
#   in this commit series, so it needs a more reasonable name.
#   
# ==============================================================================

diff -r 065a45b8659d -r a269192c5d30 layout/generic/nsGfxScrollFrame.cpp
--- a/layout/generic/nsGfxScrollFrame.cpp	Wed Jun 26 20:59:01 2019 +0000
+++ b/layout/generic/nsGfxScrollFrame.cpp	Wed Jun 26 20:59:14 2019 +0000
@@ -5409,7 +5409,7 @@
   return frame;
 }
 
-nsIFrame* ScrollFrameHelper::GetFrameForScrollSnap() const {
+nsIFrame* ScrollFrameHelper::GetFrameForStyle() const {
   nsIFrame* styleFrame = nullptr;
   if (mIsRoot) {
     if (const Element* rootElement =
@@ -5425,7 +5425,7 @@
 
 bool ScrollFrameHelper::NeedsScrollSnap() const {
   if (StaticPrefs::layout_css_scroll_snap_v1_enabled()) {
-    nsIFrame* scrollSnapFrame = GetFrameForScrollSnap();
+    nsIFrame* scrollSnapFrame = GetFrameForStyle();
     if (!scrollSnapFrame) {
       return false;
     }
@@ -6869,7 +6869,7 @@
 }
 
 nsMargin ScrollFrameHelper::GetScrollPadding() const {
-  nsIFrame* styleFrame = GetFrameForScrollSnap();
+  nsIFrame* styleFrame = GetFrameForStyle();
   if (!styleFrame) {
     return nsMargin();
   }
@@ -6886,7 +6886,7 @@
 
   ScrollSnapInfo result;
 
-  nsIFrame* scrollSnapFrame = GetFrameForScrollSnap();
+  nsIFrame* scrollSnapFrame = GetFrameForStyle();
   if (!scrollSnapFrame) {
     return result;
   }